agent/requestor/src/market.rs

```diff
@@ -28,29 +32,31 @@ pub(crate) fn build_demand(
         },
     });
 
+    let common_constraint = Constraints::new_clause(
+        And,
+        vec![
+            ConstraintKey::new("golem.inf.mem.gib").greater_than(ConstraintValue::new(0.5)),
+            ConstraintKey::new("golem.inf.storage.gib").greater_than(ConstraintValue::new(1)),
+            ConstraintKey::new("golem.com.pricing.model").equal_to(ConstraintValue::new("linear")),
+        ],
+    );
```

Comment:
This could be written using `constraints!` macro. Probably would be much more readable.